tasks/changelog.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+if RUBY_VERSION < '2.6'
+  puts 'Changelog utilities available only for Ruby 2.6+'
+  exit(1)
+end
+
+# Changelog utility
+class Changelog
+  ENTRIES_PATH = 'CHANGELOG/'
```

Comment:
I'd probably name the folder `changelog` instead, as I hate typing all caps. :D Names like `CHANGELOG` and `README` never sat well with me, but we're stuck with them for historical reasons. For the folder name we can be more flexible.